languageFactories/language-factory-core/src/main/scala/cromwell/languages/util/ImportResolver.scala

```diff
@@ -40,42 +42,46 @@ object ImportResolver {
   }
 
   object DirectoryResolver {
-    def apply(directory: Path, allowEscapingDirectory: Boolean, customName: Option[String]): DirectoryResolver = {
+    def apply(directory: Path, allowEscapingDirectory: Boolean, customName: Option[String], resolvedImportsStore: ResolvedImportsStore): DirectoryResolver = {
       val dontEscapeFrom = if (allowEscapingDirectory) None else Option(directory.toJava.getCanonicalPath)
-      DirectoryResolver(directory, dontEscapeFrom, customName)
+      DirectoryResolver(directory, dontEscapeFrom, customName, resolvedImportsStore)
     }
 
-    def localFilesystemResolvers(baseWdl: Option[Path]) = List(
+    def localFilesystemResolvers(baseWdl: Option[Path], resolvedImportsStore: ResolvedImportsStore) = List(
       DirectoryResolver(
         DefaultPathBuilder.build(Paths.get(".")),
         allowEscapingDirectory = true,
-        customName = None
+        customName = None,
+        resolvedImportsStore
       ),
       DirectoryResolver(
         DefaultPathBuilder.build(Paths.get("/")),
         allowEscapingDirectory = false,
-        customName = Some("entire local filesystem (relative to '/')")
+        customName = Some("entire local filesystem (relative to '/')"),
+        resolvedImportsStore
       )
     ) ++ baseWdl.toList.map { rt =>
       DirectoryResolver(
         DefaultPathBuilder.build(Paths.get(rt.toAbsolutePath.toFile.getParent)),
         allowEscapingDirectory = true,
-        customName = None
+        customName = None,
+        resolvedImportsStore
       )
     }
   }
 
   case class DirectoryResolver(directory: Path,
                                dontEscapeFrom: Option[String] = None,
                                customName: Option[String],
+                               override val resolvedImportsStore: ResolvedImportsStore,
```

Comment:
Is the `override` keyword strictly necessary here? I've never quite understood this nook of Scala